### DISCUSSION TITLE
Is possible to let LSPosed work without magisk?

### DISCUSSION THREAD

**Discussion by fgy58963** [LSPosed/LSPosed#1972] [Q&A]
Hi, Is that possible to install LSPosed without magisk and worked?
the Riru is not really reuired magisk. And LSPosed is depency on riru.
I think it`s possible to maunlly install LSPosed without magisk. But I am not sure it can work without magisk.
Can anyone give me an advice?
Thanks a lot

**Comment by dda99x** [comment]
Try [KernelSU](https://github.com/tiann/KernelSU) and [ZygiskOnKernelSU](https://github.com/Dr-TSNG/ZygiskOnKernelSU)
